c7n/cli.py

```diff
@@ -98,6 +98,9 @@ def _default_options(p, exclude=[]):
             help="Cache validity in minutes (default %(default)i)")
     else:
         p.add_argument("--cache", default=None, help=argparse.SUPPRESS)
+    if 'session-policy' not in exclude:
```

Comment:
We could potentially name it something else if the goal is to use it for all providers